docs/pages/api-docs/menu-list.md

```diff
@@ -31,6 +31,7 @@ the focus is placed inside the component it is fully keyboard accessible.
 | <span class="prop-name">autoFocusItem</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, will focus the first menuitem if `variant="menu"` or selected item if `variant="selectedMenu"` |
 | <span class="prop-name">children</span> | <span class="prop-type">node</span> |  | MenuList contents, normally `MenuItem`s. |
 | <span class="prop-name">disableListWrap</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, the menu items will not wrap focus. |
+| <span class="prop-name">enableFocusForDisabledItems</span> | <span class="prop-type">bool</span> | <span class="prop-default">true</span> | If `true`, will allow focus on disabled items. |
```

Comment:
For boolean props we want have `false` as the default value to enable boolean shorthands like `<MenuList disableFocusForDisabledItems />`. See https://material-ui.com/guides/api/#property-naming

Reply:
seems like the consensus is to have this turned off by default so I just updated the current prop default.